network/nxos/nxos_config.py

```diff
@@ -93,7 +93,7 @@
         without first checking if already configured.
     required: false
     default: false
-    choices: BOOLEANS
+    choices: [ "yes", "no" ]
```

Comment:
This is inconsistent with the rest of the changes.  I think generally speaking, we try to use `yes/no` instead of `true/false` anyway.  So I would consider this one correct.